content/blog/2023-07-09-GRC-standalone-week-05/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "GSoC Standalone GRC : week-5"
+author: "Rahul Balaji"
+date: "2023-07-9"
```

Comment:
misses the "0"  before the "9", i.e. *must* be `"2023-07-09"`, otherwise doesn't show up on website.